src/main/kotlin/no/nav/syfo/sykmelding/integration/aktor/client/AktoerIdClient.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.syfo.sykmelding.integration.aktor.client
+
+import io.ktor.client.HttpClient
+import io.ktor.client.call.receive
+import io.ktor.client.request.accept
+import io.ktor.client.request.get
+import io.ktor.client.request.headers
+import io.ktor.client.request.parameter
+import io.ktor.client.response.HttpResponse
+import io.ktor.http.ContentType
+import io.ktor.util.KtorExperimentalAPI
+import no.nav.syfo.client.StsOidcClient
+import no.nav.syfo.helpers.retry
+import no.nav.syfo.sykmelding.integration.aktor.model.IdentInfoResult
+import no.nav.syfo.sykmelding.util.LoggingMeta
+
+@KtorExperimentalAPI
+class AktoerIdClient(
+    private val endpointUrl: String,
+    private val stsClient: StsOidcClient,
+    private val httpClient: HttpClient,
+    private val serviceUserName: String
+) {
+    suspend fun getAktoerIds(
+        personNumbers: List<String>,
+        loggingMeta: LoggingMeta
+    ): Map<String, IdentInfoResult> =
```

Comment:
Egentlig hadde det vært fint hvis dette metodekallet returnerte en aktørid fremfor en map. Sjekk hvordan syfosmmottak eller andre apper velger riktig aktørid :)

Reply:
Godt tips, takk!

Reply:
Jeg så nå at jeg hadde stjelt dette fra syfosmmottak. Jeg tror kanskje AktørID-apiet returnerer et map med personnummer som nøkkel og et IdentInfoResult-objekt som verdi, og at det er derfor det er representert slik.

Reply:
Ja, det stemmer. Samtidig er det ålreit for de andre klassene som skal bruke denne klienten at de kun får en aktørid-string tilbake :) Se gjerne på f.eks. https://github.com/navikt/syfosmpapirmottak/blob/master/src/main/kotlin/no/nav/syfo/client/AktoerIdClient.kt